apis/connection.js

```diff
@@ -20,34 +20,34 @@
  *
  * @param {ModbusRTU} Modbus the ModbusRTU object.
  */
-var addConnctionAPI = function(Modbus) {
+var addConnctionAPI = function (Modbus) {
```

Comment:
hi,

can yo clean up the changes that are not really changes ?
like this extra space, and the extra spaces in line 27..56, they make it harder to understand what is going on here :-)